### DISCUSSION TITLE
I think this is a bug in the readme

### DISCUSSION THREAD

**Discussion by kforcodeai** [milvus-io/milvus#16202] [Q&A and General discussion]
I think this is a bug in the readme,
[this](https://github.com/milvus-io/bootcamp/blob/master/benchmark_test/lab2_sift1b_100m.md) script says insert data and then build index.
whereas [this](https://github.com/milvus-io/bootcamp/tree/master/benchmark_test/scripts) says to build index first and then insert. 
Which one is correct or efficient, inserting and then building sounds correct to me.

**Comment by xiaofan-luan** [comment]
> I think this is a bug in the readme, [this](https://github.com/milvus-io/bootcamp/blob/master/benchmark_test/lab2_sift1b_100m.md) script says insert data and then build index. whereas [this](https://github.com/milvus-io/bootcamp/tree/master/benchmark_test/scripts) says to build index first and then insert. Which one is correct or efficient, inserting and then building sounds correct to me.

Both would work~~~
For search efficiency I would like to recomment insert then build index. But there is no much diffenrent
